metrics/prometheus_exporter_test.go

```diff
@@ -68,6 +68,7 @@ func expectPromSrv(t *testing.T, expectedAddr string) {
 	}
 }
 
+// nolint // TODO: Test is currently skipped.
```

Comment:
Add the issue number to unskip (for grep)?

Reply:
Inlined it instead, even prettier.